client/a8c-for-agencies/sections/referrals/primary/referrals-overview/index.tsx

```diff
@@ -174,6 +182,28 @@ export default function ReferralsOverview() {
 										onClick: onDownloadA4APluginClick,
 									} }
 								/>
+								<StepSectionItem
+									className="referrals-overview__step-section-woo-payments"
+									icon={ <WooLogo /> }
+									heading={ translate( 'Install WooPayments on your clients’ online stores' ) }
+									description={
+										<>
+											{ translate(
+												'Receive a revenue share of 5 basis points (0.05%) on new WooPayments gross merchandise value on clients’ sites.'
+											) }
+											<div>
```

Comment:
Is the extra div here necessary to achieve a specific style?

Reply:
Yes. To keep the button on the next line.